cms/static/js/views/settings/main.js

```diff
@@ -151,6 +162,22 @@ var DetailsView = ValidatingView.extend({
         case 'course-effort':
             this.setField(event);
             break;
+        case 'entrance-exam-enabled':
+            if($(event.currentTarget).is(":checked")){
+                this.$el.find('#div-grade-requirements').show();
```

Comment:
We have a policy of not using ids unless they are absolutely necessary. I'd make this be a class instead.